lib/clients/copyFactory/trading.client.d.ts

```diff
@@ -2,7 +2,7 @@ import DomainClient from "../domain.client";
 import MetaApiClient from "../metaApi.client";
 import { CopyFactoryStrategyIdAndName } from "./history.client";
 import SignalClient from "./signal.client";
-import StopoutListener from "./stopoutListener";
+import StopoutListener from './streaming/stopoutListener';
```

Comment:
Please, follow project eslint rooles. 
You can do check by using project script: "lint".

Replace please single quotes with double quotes at line 5.